packages/mds-stream/nats/agency-stream-nats.ts

```diff
@@ -0,0 +1,23 @@
+import { VehicleEvent, Telemetry, Device } from '@mds-core/mds-types'
+import { getEnvVar } from '@mds-core/mds-utils'
+import { AgencyStreamInterface } from '../agency-stream-interface'
+import { NatsStreamProducer } from './stream-producer'
+
+const { TENANT_ID } = getEnvVar({
+  TENANT_ID: 'mds'
+})
+const deviceProducer = NatsStreamProducer<Device>(`${TENANT_ID}.device`)
+const eventProducer = NatsStreamProducer<VehicleEvent>(`${TENANT_ID}.event`)
+const telemetryProducer = NatsStreamProducer<Telemetry>(`${TENANT_ID}.telemetry`)
+
+export const AgencyStreamNats: AgencyStreamInterface = {
+  initialize: async () => {
+    await Promise.all([deviceProducer.initialize(), eventProducer.initialize(), telemetryProducer.initialize()])
+  },
+  writeEvent: eventProducer.write,
```

Comment:
We can probably use the `...handlers` pattern here, right?

Reply:
Probably -- I'm going to leave that for a second refactor just so I can get this PR in.